CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## 0.13.0 (2024-02-05)
+
+- Make `ParseError` and `ValidationError` public
```

Comment:
This needs to be added to the `0.12.0` entry as that is the next version to be released